Algorithm/QCAlgorithm.Python.cs

```diff
@@ -396,29 +396,19 @@ public void RegisterIndicator(Symbol symbol, PyObject indicator, IDataConsolidat
         /// <seealso cref="Plot(string,decimal)"/>
         public void Plot(string series, PyObject pyObject)
         {
-            IIndicator<IndicatorDataPoint> indicator;
-
             using (Py.GIL())
             {
-                var pythonType = pyObject.GetPythonType();
-
                 try
                 {
-                    var type = pythonType.As<Type>();
-                    indicator = pyObject.AsManagedObject(type) as IIndicator<IndicatorDataPoint>;
-
-                    if (indicator == null)
-                    {
-                        throw new ArgumentException();
-                    }
+                    decimal value = ((dynamic)pyObject).Current.Value;
```

Comment:
Nice!
Would be great if we can add some tests covering the reported issue or similar potential issue